packages/piper/src/fsutils.ts

```diff
@@ -97,12 +98,16 @@ export async function runJSFunction(
       }
     };
 
-    (process.stdout.write as any) = (chunk: any) => {
+    (process.stdout.write as any) = (chunk: any, enc?: any, cb?: any) => {
```

Comment:
we don't like the any types here, and... we know what these types should be... a buffer or a stirng... we hate all of this any type! god damn it you guys.